src/spikeinterface/preprocessing/whiten.py

```diff
@@ -223,27 +198,88 @@ def compute_whitening_matrix(
     # type and we estimate a more reasonable eps in the case
     # where the data is on a scale less than 1.
     if eps is None:
-        eps = 1e-8
-    if data.dtype.kind == "f":
```

Comment:
As the `random_chunk` -> `data` is always float, I removed the `int16` option, let me know if this makes sense.

Out of interest, what is the purpose of this scaling? will it have much effect if it is capped in the maximum direction by `1e-16`?